Use defusedxml.

jjnicola opened the pull request:
Use defusedxml.fromstring instead of xml.etree.ElementTree.fromstring
to parse unstrusted XML data, since it is know to be vulnerable to XML
attacks.

janowagner commented:
Although it is not a security problem for operation, the tests in tests/testScanAndResult.py should
also be converted to the secET.
This would not only make CI tests happy. It would also reduce chance to have developers
copy the code for other uses where security matters.

jjnicola committed:
Use defusedxml.
Use defusedxml.ElementTree.fromstring instead of xml.etree.ElementTree.fromstring
to parse unstrusted XML data, since it is know to be vulnerable to XML
attacks.

jjnicola commented:
I have a patch for the tests in tests/testScanAndResult.py. I can not commit it with this PR because it will have conflicts with the PR #31. Therefore it will be committed on a new PR.

janowagner reviewed (approved):
OK, so lets go with this change and let follow the change to the tests for secure XML parsing.
